tests/NoSandboxCrawler.php

```diff
@@ -0,0 +1,152 @@
+<?php
+
+namespace Spatie\Crawler\Test;
+
+use Tree\Node\Node;
+use GuzzleHttp\Pool;
+use GuzzleHttp\Psr7\Uri;
+use Spatie\Crawler\Crawler;
+use Spatie\Crawler\CrawlUrl;
+use InvalidArgumentException;
+use Psr\Http\Message\UriInterface;
+use Spatie\Crawler\CrawlSubdomains;
+use Symfony\Component\DomCrawler\Link;
+use Psr\Http\Message\ResponseInterface;
+use Tightenco\Collect\Support\Collection;
+use GuzzleHttp\Exception\RequestException;
+use Symfony\Component\DomCrawler\Crawler as DomCrawler;
+
+class NoSandboxCrawler extends Crawler
```

Comment:
Even though technically this makes the tests pass, this solution is not maintainable.

If we change something in the crawler class, there's a high change that we'll forget to update it here.

Instead of coping all the code, we should introduce a way to make the browsershot instance configurable from the outside.

First thought would be to add let the `executeJavaScript` function optionally accept an instance or Browsershot or a closure where in a user can configure an instance of browsershot.

That way we can can configure Browsershot in our tests to our hearts content and we can remove all this whole `NoSandboxCrawler` class.